### PR TITLE
Mark/doc comments

### PR DESCRIPTION
**Pull request by marklnichols**

### CODE REVIEW

**Comment by marklnichols** [comment]
I merged the mark/multi-baker branch into this one

File: README.md [hunk 1]
```diff
@@ -11,24 +11,204 @@ This doc will describe how to run your own private Tezos blockchain.
...
+
+We will use the alias 'alice' to refer to the bootstrap_accounts entry with these values:
+```sh
+Hash: tz1akcPmG1Kyz2jXpS4RvVJ8uWr7tsiT9i6A
+Public Key: edpktezaD1wnUa5pT2pvj1JGHNey18WGhPc9fk9bbppD33KNQ2vH8R
+Secret Key: unencrypted:edsk2vKVH2BNwKrxJrvbRvuHnu4FW17Jrs2Uy2TzR2fxipikTJJ1aG
+```
+Enter the following command:
+```sh
+$ tezos-client import secret key alice unencrypted:edsk2vKVH2BNwKrxJrvbRvuHnu4FW17Jrs2Uy2TzR2fxipikTJJ1aG
```
Comment (rvem) [inline README.md:148]: I think it worth saying that one can use `tezos-client` which is located in the container or obtain one from https://github.com/serokell/tezos-packaging

File: scripts/start-baker.sh [hunk 2]
```diff
@@ -66,6 +66,7 @@ usage() {
     echo "  --tezos-node <filepath>. Path for patched tezos-node executable"
     echo "  --tezos-baker <filepath>. Path for patched tezos-baker executable"
     echo "  --tezos-endorser <filepath>. Path for patched tezos-endorser executable"
+    echo "  --rpc-addr <rpc-addr>. Define RPC address of the baker node"
```
Comment (rvem) [inline scripts/start-baker.sh:69]: Still insist on making this argument optional (just like `--peer`) and use `localhost:8732` as default. You can offload this change to me if you want to

**Comment by rvem** [comment]
Generally, looks very good :+1:

File: README.md (hunk 1 above)
Comment (marklnichols) [reply README.md:148]: Ok, done.

File: scripts/start-baker.sh (hunk 2 above)
Comment (marklnichols) [reply scripts/start-baker.sh:69]: It does seem to be optional now

**Comment by marklnichols** [comment]
This PR is now replace by #13

**Pull request closed without merging**